client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -150,8 +161,8 @@ export default ({ id, siteId, onArchiveParticipantAction }) => {
             participantName: `${row.participantJoin.body.firstName} ${row.participantJoin.body.lastName}`,
             hiredDate: row.data.hiredDate,
             startDate: row.data.startDate,
-            withdrawnDate: row.created_at,
-            reason: row.data.final_status,
+            withdrawnDate: row.data.endDate,
```

Comment:
possibly data is integral part of row object, but for safety we can use optional chaining.
like ``
```suggestion
            withdrawnDate: row.data?.endDate,
```